man/shadow.5.xml

```diff
@@ -72,15 +72,15 @@
 	<term><emphasis role="bold">encrypted password</emphasis></term>
 	<listitem>
 	  <para>
-	    This field may be empty, in which case no passwords are
-	    required to authenticate as the specified login name.
-	    However, some applications which read the
-	    <filename>/etc/shadow</filename> file may decide not to permit
-	    any access at all if the password field is empty.
+	    This field may be empty, in which case
+	    no passwords are required to authenticate with the specified login name.
```

Comment:
@hallyn , I'm not native.  What sounds better to you?  'as' or 'with'?

Reply:
'as'